### ISSUE TITLE
Deprecation Warnings for autoloading during initialization

### ISSUE THREAD

**Issue by fosterfarrell9** [MaMpf-HD/mampf#85]
There is a deprecation warning during initializiation:

```
DEPRECATION WARNING: Initialization autoloaded the constants ApplicationRecord, ApplicationHelper, QuizGraph, Solution, VideoUploader, ScreenshotUploader, PdfUploader, GeogebraUploader, HttpUrlValidator, and Medium.

 Being able to do this is deprecated. Autoloading during initialization is going
to be an error condition in future versions of Rails.

 Reloading does not reboot the application, and therefore code executed during
 initialization does not run again. So, if you reload ApplicationRecord, for example,
 the expected changes won't be reflected in that stale Class object.

These autoloaded constants have been unloaded
```

We should fix that as that will trigger an error in Rails 6.1.

**Comment by henrixapp** [comment]
According to https://medium.com/@dylansreile/rails-6-0-new-framework-defaults-what-they-do-and-how-to-safely-uncomment-them-586146f371e8 we need to require classes explicitly if used in Initializers

**Comment by fosterfarrell9** [comment]
Closed via #273
